libs/island-ui/core/src/lib/Checkbox/Checkbox.tsx

```diff
@@ -79,7 +82,12 @@ export const Checkbox = ({
             color={checked ? 'white' : 'transparent'}
           />
         </div>
-        {label}
+        <span className={styles.labelText}>
```

Comment:
Consider using the Text component here instead of a span. That way, any changes made to the Typography in the future are reflected here as well. You can use `as="span"` to render Text as a span.

Reply:
Yes, I will use the `<Text>` component for the `subLabel` but I will keep this span as is since it's only used as a container for the `label` and `subLabel` and no text styles are applied.